File: RxDiRetrofitApp/japp/src/main/java/com/woowrale/rxdiretrofit/ui/main/MainView.java
package com.woowrale.rxdiretrofit.ui.main;

import com.woowrale.rxdiretrofit.data.model.Crypto;
import com.woowrale.rxdiretrofit.ui.base.BasePresenter;


import java.util.List;

public interface MainView extends BasePresenter.View {

    void showListView(List<Crypto.Market> marketList);

}

File: RxDiRetrofitApp/kapp/src/main/java/com/woowrale/rxdiretrofit/ui/main/MainView.kt
package com.woowrale.rxdiretrofit.ui.main

import com.woowrale.rxdiretrofit.data.model.Market
import com.woowrale.rxdiretrofit.ui.base.BasePresenter


interface MainView: BasePresenter.View {
    fun setRecyclerViewAdapterData(marketList: List<Market>)
}